plugins/examples/painless-whitelist/build.gradle

```diff
@@ -33,7 +35,7 @@ dependencies {
 }
 
 testClusters.integTest {
-  testDistribution = 'oss'
+  testDistribution = BuildParams.inFipsJvm ? 'DEFAULT' : 'oss'
```

Comment:
Why the different casing?

Reply:
Groovy is lenient here. I've made this consistently uppercase.